app-route.html

```diff
@@ -236,9 +237,8 @@
 
       __resetProperties: function() {
```

Comment:
Perhaps rename to `__deactivate`?

If we do end up going to this PR, perhaps we should provide a public `resetProperties` or `clearProperties` helper function that clears `data` and `tail`. This function not called by `__deactivate`, so the user can choose the functionality. By changing the path and tail, that should eventually do the same as [this other proposed solution](https://github.com/PolymerElements/app-route/compare/force-route-state-change).

Thoughts @cdata?